frontend/main/hideNavbar.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+(function () {
+    const app = angular.module('app');
+
+    app.directive('hideNavbar', function(STATES, $state) {
```

Comment:
use inline array annotation here and in all dependency injectors
https://docs.angularjs.org/guide/di#inline-array-annotation